cmd/sidecars/sidecar_shim.go

```diff
@@ -195,6 +195,7 @@ func runPreCloudInitIso(vmiJSON []byte, cloudInitDataJSON []byte) ([]byte, error
 
 	log.Log.Infof("Executing %s", preCloudInitIsoBin)
 	command := exec.Command(preCloudInitIsoBin, args...)
+	command.Stderr = os.Stderr
```

Comment:
Does this really solve the issue? Are we printing the stderr somewhere? Why not using the Combinedoutput as Itamar suggested below?

Reply:
Is.stderr is the K8s logs. 
Combined output just makes it harder to read and I don't understand the benefit.

Reply:
I find it a bit weird that we only print the stderr on the pod log and not the std out. Do you have an example how this look like in the pod output log if there is a failure? 
For example, does it only print the raw error message of the cloud init, or do we provide also some context.?

Reply:
Printing to `stderr` would show in the `virt-launcher` logs, not structured though (not wrapped by `sidecar-shim` as component for example)

Reply:
The `stdout` of the script is used to output the `domain xml` on `OnDefineDomain` hook. We might break things if we combine both stdout and stderr in a single stream.

Reply:
@victortoso my only concern is that the log would be meaningless. For example, if we don't give the command that caused the error. Also the fact that we don't wrapped it as caused by the shim seems problematic to me.
Do you think is still better to have like this rather then nothing?

Reply:
/lgtm cancel

No, I think we should have the stderr logs wrapped.
We need a way to log from the scripts still. The easiest I think is using stderr.